launcher/internal/oci/interface.go

```diff
@@ -0,0 +1,71 @@
+// Package oci contains functionalities to interact with OCI image signatures.
+// https://github.com/opencontainers/image-spec/tree/main#readme.
+package oci
+
+import (
+	"go.uber.org/multierr"
+)
+
+// SigningAlgorithm is a specific type for string constants used for sigature signing and verification.
+type SigningAlgorithm string
+
+const (
+	// RsassaPss2048Sha256 is RSASSA-PSS 2048 bit key with a SHA256 digest supported for cosign sign.
+	RsassaPss2048Sha256 SigningAlgorithm = "RSASSA_PSS_2048_SHA256"
```

Comment:
nit: I believe golang style is to capitalize acronyms, so RSASSAPSS2048SHA256 (as painfully loud as that looks haha)

Reply:
Done.

Reply:
I think in golang we do prefer the CamelCase. Like in the standard library https://cs.opensource.google/go/go/+/master:src/io/io.go;l=22 and in this repo as well.

Reply:
Ah I see they are acronyms, then all caps should be good.